packages/core/src/Chip/ChipContentWrapper.tsx

```diff
@@ -31,6 +31,18 @@ const getBorderColor = (props) =>
 const getBackgroundColor = (props) =>
   props.disabled ? 'background.base' : props.selected ? 'light' : 'background.lightest'
 
+const variations = {
+    outline: css` 
+    `,
+    shadow: css`
+      border-color: transparent;
```

Comment:
@wesleymartin85 

> the border here should be set to the same color as the white background. (this is just for the default enabled state the primary.dark border on hover is correct. 

Are you asking to make the border-color exactly what the background color is in the stories, or make it transparent so it's what ever color the background is?

Reply:
its not for the storyies but part of the variation settings, look that the styles in the [prototype](https://www.figma.com/proto/gpbXA3aonRBAJbQiZvrayO/Design-Ops-Dev-Week?page-id=0%3A1&type=design&node-id=564-22482&viewport=-25300%2C2359%2C1&scaling=min-zoom&starting-point-node-id=564%3A22482&t=eLmmLiEs6OFOYLsR-1&show-proto-sidebar=1) and [designs](https://www.figma.com/file/gpbXA3aonRBAJbQiZvrayO/Design-Ops-Dev-Week?type=design&node-id=215-7397&mode=design&t=0gi2R1mn5lWSAqlS-11)

Reply:
Yea, I'm just uncertain if you want the border-color to be a static color or should it be the color of whatever the background is that the chip is on top of.  I think we want it to be transparent so it's whatever the background color is, but not 100% sure.   Currently it's set to transparent, so let me know how that looks in storybook.